minion.conf

```diff
@@ -0,0 +1,11 @@
+file_client: local
+
+fileserver_backend:
+  - git
+  - roots
+
+gitfs_provider: gitpython
+
+gitfs_remotes:
+  - https://github.com/mitodl/salt-extensions:
+      - root: extensions
```

Comment:
This file shouldn't even be part of this. Think it's generated when I ran the vagrantfile. Will remove it in next commit.